### PR TITLE
Harmonize margins within the relations drag-and-drop designer settings

### PR DESCRIPTION
**Pull request by DelazJ**
Still ugly and crowded but a harmonized ugliness
before (from 3.22)
image
after (master)
image

### CODE REVIEW

File: src/gui/attributeformconfig/qgsattributewidgetedit.cpp [hunk 1]
```diff
@@ -37,6 +37,7 @@ QgsAttributeWidgetEdit::QgsAttributeWidgetEdit( QTreeWidgetItem *item, QWidget *
     case QgsAttributesFormProperties::DnDTreeItemData::Relation:
     {
       QGridLayout *layout = new QGridLayout;
+      layout->setContentsMargins( 0, 0, 0, 0);
```
Comment (DelazJ) [inline src/gui/attributeformconfig/qgsattributewidgetedit.cpp:40]:
```suggestion
      layout->setContentsMargins( 0, 0, 0, 0 );
```

**Review by nyalldawson — APPROVED** [review]

**Comment by nyalldawson** [comment]
Thank you!

> Still ugly and crowded but a harmonized ugliness

and yes... ugly indeed :scream:

**Comment by nyalldawson** [comment]
Hmm

couldn't we:

1. move the "toolbar buttons" group out to sit on its own
2. move "widget configuration" group out to its own, and move the "force hide form " checkbox into that
3. 
4. ?

**Comment by nyalldawson** [comment]
4. Fix capitalisation of "Toolbar buttons" (should be title case)
5. move to "show lable" checkbox to the widget configuration gruop
6. move the two override group boxes so they aren't nested

**Pull request closed without merging**